packages/amplication-build-manager/.env

```diff
@@ -30,4 +30,4 @@ DSG_JOBS_RESOURCE_DATA_FILE="input.json"
 DSG_RUNNER_URL="http://localhost:8900/api/code-generate"
 
 # DSG catalog service
-DSG_CATALOG_SERVICE_URL="https://data-service-generator-catalog.amplication.com/"
+DSG_CATALOG_SERVICE_URL="https://data-service-generator-catalog.amplication-sandbox.com/"
```

Comment:
I don't really mind this since this is overridden from cluster-config anyway. But it's better to either not commit this or to do this with a .env.local